src/test/java/tests/AccountTest.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class AccountTest extends BaseTest {
+
+    @Test
+    public void createAccount() {
+        loginPage.openPage()
+                .waitForPageOpened()
+                .login(USERNAME, PASSWORD)
+                .waitForPageOpened();
+        accountListPage.openPage()
+                .waitForPageOpened()
+                .clickNewBtn()
+                .waitForPageOpened()
+                .create(account)
+                .waitForPageOpened()
+                .openDetailsTab()
+                .waitForTabOpened();
+        String accountNameActual = accountDetailsPage.getAccountDetails("Account Name");
+        String typeActual = accountDetailsPage.getAccountDetails("Type");
+        String descriptionActual = accountDetailsPage.getAccountDetails("Description");
+        String employeesActualStr = accountDetailsPage.getAccountDetails("Employees");
+        Integer employeesActual = Integer.parseInt(employeesActualStr);
+        String industryActual = accountDetailsPage.getAccountDetails("Industry");
+        String phoneActual = accountDetailsPage.getAccountDetails("Phone");
+        String websiteActual = accountDetailsPage.getAccountDetails("Website");
```

Comment:
It's okay. But there is also another way to put all these strings. You create Account object (for i.e. actualAccount) and create one method which set all these fields in Account object. You can leave this 'as is' or try to implement another approach, up to you :)

Reply:
Yes, I can but here I'm creating strings as a actual result to be passed to assertEquals therefore if i create an Account object and pass it here I still need to create strings as actual result for assert anyway